sanic/response.py

```diff
@@ -319,9 +319,30 @@ def html(
     )
 
 
+async def validate_file(
+    request_headers: Header, last_modified: Union[datetime, float, int]
+):
+    try:
+        if_modified_since = request_headers.getone("If-Modified-Since")
+    except KeyError:
+        return
+    try:
+        if_modified_since = parsedate_to_datetime(if_modified_since)
+    except (TypeError, ValueError):
```

Comment:
Let's add:

```suggestion
    except (TypeError, ValueError):
		logger.warning(
			"Ignorning invalid If-Modified-Since header received: "
            "'%s'",
            if_modified_since
        )
```

Reply:
Thanks!